public/app/features/dashboard/dashgrid/PanelChrome.tsx

```diff
@@ -89,53 +88,16 @@ export class PanelChrome extends PureComponent<Props, State> {
         onAnnotationCreate: this.onAnnotationCreate,
         onAnnotationUpdate: this.onAnnotationUpdate,
         onAnnotationDelete: this.onAnnotationDelete,
-        canAddAnnotations: this.canAddAnnotation,
         onInstanceStateChange: this.onInstanceStateChange,
         onToggleLegendSort: this.onToggleLegendSort,
-        canEditAnnotations: this.canEditAnnotation,
-        canDeleteAnnotations: this.canDeleteAnnotation,
+        canAddAnnotations: props.dashboard.canAddAnnotations.bind(props.dashboard),
+        canEditAnnotations: props.dashboard.canEditAnnotations.bind(props.dashboard),
+        canDeleteAnnotations: props.dashboard.canDeleteAnnotations.bind(props.dashboard),
       },
       data: this.getInitialPanelDataState(),
     };
   }
 
-  canEditDashboard = () => Boolean(this.props.dashboard.meta.canEdit || this.props.dashboard.meta.canMakeEditable);
```

Comment:
I have to remove the 3 methods `canAddAnnotation`, `canEditAnnotation`, and `canDeleteAnnotation` as these are duplicated logic from `DashboardModel.ts`